### PR TITLE
Release 0.3.3

### PR DESCRIPTION
**Pull request by 928PJY**
Depends on #230

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,24 @@ This extension enables you to run build validation on a Docs conceptual or Learn
 
 - Install [git](https://git-scm.com/downloads).
 - Clone your Docs or Learn repo locally in VS Code.
-- All files in the repo must be saved.
+- All files in the repo must be saved if you trigger the manual validation.
 
 ## How to use the Docs Validation extension
 
-### Full-repository validation
+### Manual validation
```
Comment (shanluo911) [inline README.md]: Not sure, but I think maybe 'manual(ly) triggered validation' is better?

**Review by herohua — CHANGES_REQUESTED** [review]
Hold off this release until PM confirms

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 0.3.3 (March 25, 2021)
+
+### Added
+
+- Support Activating extension when the user is on the sub-folder of Docs repository. (#209)
```
Comment (shanluo911) [inline CHANGELOG.md]: typo lower case `a`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+- Support trigger validation on the sub-folder of the current repository. (#229)
```
Comment (shanluo911) [inline CHANGELOG.md]: triggering

File: README.md [hunk 4]
```diff
@@ -8,16 +8,24 @@ This extension enables you to run build validation on a Docs conceptual or Learn
...
 
-1. Open a Docs conceptual or Learn repo in VS Code with the extension installed.
+You can trigger the validation manually with the following options:
```
Comment (shanluo911) [inline README.md:17]: remove `the`

File: README.md [hunk 5]
```diff
@@ -8,16 +8,24 @@ This extension enables you to run build validation on a Docs conceptual or Learn
...
+
+- Full repository validation
+- Sub-repository validation
+
+1. Open the root folder or any sub-folder of a Docs conceptual or Learn repo in VS Code with the extension installed.
 1. For the first time you use the extension, you will be asked to choose your user type between **Microsoft employee** and **Public contributor** before you can use full-repository validation.
 1. For **Microsoft employee**, you are required to sign in before using full-repository validation. You can click **Docs Validation** on the status bar then click **Sign in** from the drop-down menu to sign in.
-1. After signing in, you can trigger a validation by clicking **Validate** when prompted. You can also click **Docs Validation** on the status bar then click **Validate** from the drop-down menu, or by right-clicking any file in the repo and selecting **Validate this repository**.
+1. You three ways to trigger the manual validation:
```
Comment (shanluo911) [inline README.md]: there are

File: README.md [hunk 6]
```diff
@@ -8,16 +8,24 @@ This extension enables you to run build validation on a Docs conceptual or Learn
...
+   1. After signing in, you can trigger a validation by clicking **Validate** when prompted. This will trigger the validation on the current opening workspace, so the files outside the current workspace will not be validated if you are on the sub-folder of a repository.
+   2. You can click **Docs Validation** on the status bar then click **Validate the current workspace** or **Validate the whole repository** from the drop-down menu to trigger the validation on different scope.
```
Comment (shanluo911) [inline README.md]: scopes

File: README.md [hunk 7]
```diff
@@ -8,16 +8,24 @@ This extension enables you to run build validation on a Docs conceptual or Learn
...
+   3. You can right-clicking any file/folder inside the **Explorer** and selecting **Validate this folder**, if you click on the file, the validation will be applied to the folder contains the selected file, otherwise, the validation will be applied to the selected folder.
```
Comment (shanluo911) [inline README.md]: right-click and select

**Review by shanluo911 — APPROVED** [review]

**Pull request merged**